### PR TITLE
Implement Playable Community Game On HomePage

### PR DESCRIPTION
**Pull request by DevIos01**
as requested on #1568, I have implemented so that it takes the random game from "New Tag" and renders the game

### CODE REVIEW

**Comment by grymmy** [comment]
@DevIos01 please revisit the linked-to issue and get product mgmt approval (retroactively) for this work

**Review by grymmy — CHANGES_REQUESTED** [review]
see prev. comment

File: src/pages/index.astro [hunk 1]
```diff
@@ -28,6 +32,29 @@ if (session && session.session.full) {
 		.get()
 	gameCount = res.data().count
 }
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+async function getGameSourceCode() {
+	const response = await fetch("http://localhost:3000/api/gallery?new");
```
Comment (JosiasAurel) [inline src/pages/index.astro:40]: this won't work in production will it?

File: src/pages/index.astro [hunk 2]
```diff
@@ -294,15 +321,14 @@ if (session && session.session.full) {
 				})
 			}
 		</script>
-
+		<div id='gamesource-data' data-gamesource={gamesource} style={{ displa: "none", visibility: "hidden"}}></div>
```
Comment (JosiasAurel) [inline src/pages/index.astro:324]: should be `display` and not `displa`

File: src/pages/index.astro [hunk 3]
```diff
@@ -345,10 +371,14 @@ if (session && session.session.full) {
...
 					hoveredButton: Object3D | null
 					pressedButton: Object3D | null
+					hoveredScreen: Object3D | boolean
+					pressedScreen: Object3D | boolean
 				} = {
 					device: null,
 					hoveredButton: null,
 					pressedButton: null,
+					hoveredScreen: false,
+					pressedScreen: false,
```
Comment (JosiasAurel) [inline src/pages/index.astro:381]: what do these changes do?

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

**Comment by grymmy** [comment]
@DevIos01 Do you intend on moving forward with this?  If not, please close it.

**Comment by grymmy** [comment]
actually nevermind... I don't feel a strong need for this feature period.  Closing it myself, DM me if u really want to advocate for this

**Pull request closed without merging**